aries_vcx/src/indy/wallet.rs

```diff
@@ -33,6 +33,8 @@ pub struct WalletConfig {
     pub rekey: Option<String>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub rekey_derivation_method: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub master_secret: Option<String>,
```

Comment:
- This structure is currently shared for creating and opening wallets, however the `master_secret` would only be used with call `create_wallet_with_master_secret`
- Putting it into the structure would indicate that one wallet can only have 1 master secret. Yes, that's kind of the default right now anyway with the `DEFAULT_LINK_SECRET_ALIAS`, but since we are at it, let's not bake-in this "singleton-ism" into `WalletConfig` structure
- Instead I suggest to change signature of 
```
create_wallet_with_master_secret(config: &WalletConfig)
```
to 
```
create_wallet_with_master_secret(config: &WalletConfig, master_secret_alias: String)
```